src/app/marathon/schedule-management/edit/schedule-table/normal-run-editor/normal-run-editor.component.html

```diff
@@ -0,0 +1,143 @@
+<div class="field is-horizontal">
+  <div class="field-label is-normal">
+    {{ 'marathon.schedule.table.runner' | translate}}:
+  </div>
+  <div class="field-body">
+    <ul>
+      <li *ngFor="let runner of line.runners; let ri = index">
+        <a href="#" class="userAction"
+           *ngIf="line.runners.length > 1 || line.customRun"
+           (click)="removeUser(ri)"
+        >
+          <fa-icon [icon]="iconTimes"></fa-icon>
+        </a>
+        <app-user-link
+          [user]="runner"
```

Comment:
This would probably allow reverting the changes made this the component
```suggestion
          [user]="runner.profile"
```

Reply:
Needs to be fixed later